benchexec/tablegenerator/test_integration/__init__.py

```diff
@@ -12,16 +12,22 @@
 import sys
 import tempfile
 import unittest
+import platform
 
 import benchexec
 import benchexec.util
+import benchexec.tablegenerator.util
 
 sys.dont_write_bytecode = True  # prevent creation of .pyc files
 
 here = os.path.relpath(os.path.dirname(__file__))
 base_dir = os.path.join(here, "..", "..", "..")
 bin_dir = os.path.join(base_dir, "bin")
-tablegenerator = os.path.join(bin_dir, "table-generator")
+tablegenerator = [os.path.join(bin_dir, "table-generator")]
+line_ending = "\n"
```

Comment:
`os.linesep` can be used instead. But instead of changing all the constants, I would normalize the tool output before comparing it against the expected value.